Tasks/AzureResourceManagerTemplateDeploymentV3/operations/DeploymentScopeBase.ts

```diff
@@ -18,7 +19,27 @@ export class DeploymentScopeBase {
     }
 
     public async deploy(): Promise<void> {
-        await this.createTemplateDeployment();
+        try {
```

Comment:
the try-catch should be moved inside createTemplateDeployment(). This way you won;t have to handle it in two places.

Reply:
Done